R/Germany.R

```diff
@@ -32,6 +32,9 @@ Germany <- R6::R6Class("Germany",
     #' @importFrom dplyr select mutate
     #' @importFrom lubridate as_date ymd_hms
     clean = function() {
+      if (self$verbose) {
+        message("Cleaning data")
```

Comment:
Just to check - do you think this is a good idea?

Reply:
Yes